include/ScanningSession.h

```diff
@@ -280,6 +291,15 @@ class ScanningSession
         /// Our own tailoring that may or may not initially be loaded from a file
         mutable struct xccdf_tailoring* mTailoring;
 
+        /// Temporary copy of opened XCCDF file
+        QTemporaryDir* mOpenDir;
+        /// Path to temporary XCCDF file
+        QString mOpenPath;
```

Comment:
What about renaming this to `pathToOpenInput` or to something longer, so you can drop the comment? Long names tell you what is the variable content without requiring you to search for the information provided in comments.